src/App.tsx

```diff
@@ -59,3 +76,161 @@ export const App = () => {
     </div>
   );
 };
+
+type State = {
+  todos: Todo[],
+  users: User[],
+  newTitle: string,
+  newUser: string,
+  titleWarning: boolean,
+  userWarning: boolean,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    todos: todosWhole,
+    users: usersFromServer,
+    newTitle: '',
+    newUser: '0',
+    titleWarning: false,
+    userWarning: false,
+  };
+
+  getId = () => {
+    let maxId = 0;
+
+    this.state.todos.map(todo => {
+      if (maxId < todo.id) {
+        maxId = todo.id;
+      }
+
+      return todo;
+    });
+
+    return maxId + 1;
+  };
+
+  verifyInput = () => {
+    if (this.state.newTitle === '') {
+      this.setState({ titleWarning: true });
+
+      return false;
+    }
+
+    if (this.state.newUser === '0') {
```

Comment:
Why do you use string for userId? Use numbers instead
```suggestion
    if (!this.state.newUser) {
```